packages/components/src/navigator/navigator-provider/component.tsx

```diff
@@ -41,8 +46,13 @@ function UnconnectedNavigatorProvider(
 		},
 	] );
 
+	const [ animationSettings, setAnimationSettings ] = useState(
+		initialAnimationSettings
```

Comment:
I wonder if exposing the animation settings like this is best option. In reality it's only the direction (`x`) of the initial animation state that we need to alter, so is it better to expose an option that tells the navigator whether this is an "up" or "down" style animation, rather than making everything customizable?